### ISSUE TITLE
Readme isn't up to date

### ISSUE THREAD

**Issue by Azurency** [Azurency/CQUI_Community-Edition#124]
<a href="https://github.com/JHCD"><img src="https://avatars3.githubusercontent.com/u/12496340?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [JHCD](https://github.com/JHCD)**
_Friday Feb 24, 2017 at 08:55 GMT_
_Originally opened as https://github.com/CQUI-Org/cqui/issues/327_

----

> NOTE: Until this mod is in a less experimental state, the mod will not remember its "enabled" status after restarting the game. This means that every time you plan to start a new game using CQUI, you'll need to explicitly enable it in the "Additional Content" menu first. Once you've created the game using the mod, it will always load when you start the save, regardless of its current status in the "Additional Content Menu"

Is this still up-to-date?
It's always _enabled_ here.

**Comment by Azurency** [comment]
<a href="https://github.com/chaorace"><img src="https://avatars2.githubusercontent.com/u/6462592?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [chaorace](https://github.com/chaorace)**
_Friday Feb 24, 2017 at 08:58 GMT_

----

No, it's no longer an up to date description. I plan to just completely rewrite the readme for the Beta release

**Comment by Azurency** [comment]
<a href="https://github.com/JHCD"><img src="https://avatars3.githubusercontent.com/u/12496340?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [JHCD](https://github.com/JHCD)**
_Sunday May 21, 2017 at 07:12 GMT_

----

We need a documentation of all used key bindings and mouse-click implementations too.
Has anybody an overview about the possible features?

See #536 and #529 for example...

**Comment by ricanuck** [comment]
Not only the readme, we should also work on the Steam WS description and make it actual, especially now that the C stands for Community. I would also make every file name person-neutral, get rid of any names and personalization so that we truly reflect the Community nature of this new edition (in-code comments with names are OK, and even better, as they help identify code authors for better troubleshooting)...

I can help with the descriptions, if you want..

**Comment by sparrow8332** [comment]
@ricanuck any help is appreciated mate. @Azurency is on the ball with the updates and i would have started a fresh readme and clearing out the files ect but been bogged down so much recently with work.

**Comment by ricanuck** [comment]
yeah, I know, work, that stupid invention that prevents us from civving freely... :rofl:

**Comment by Azurency** [comment]
Why would we remove all the names and thanks ? I think it's good to thank the people who contributed, that the community ! Maybe just make it more compact on the readme

**Comment by sparrow8332** [comment]
or maybe even a link to a Pastebin with all the names off everyone thats helped get this to where it is  Past & Present

**Comment by ricanuck** [comment]
@Azurency : Antoine, I was referring to removing names from file names... right now there are some files with user names, I meant we should make all those file names neutral... off course contributor's names should stay! But the mod itself, and any of its parts, should really be person-neutral, now that the "C" stands for COMMUNITY and not for a person... capisce? :wink:

**Comment by sparrow8332** [comment]
@ricanuck ive started on sparrow8332-patch-1 branch - CQUI.modinfo for eg  is this what you mean ?

**Comment by ricanuck** [comment]
Yep, that is what I mean... neutral file names. That one should just be CQUI.modinfo...

**Comment by ricanuck** [comment]
Comments inside the code are OK, in fact it's better to know who wrote what for easier troubleshooting.
